crates/js-sys/Cargo.toml

```diff
@@ -18,8 +18,13 @@ edition = "2018"
 test = false
 doctest = false
 
+[features]
+rust-num = ["num-bigint", "num-traits"]
+
 [dependencies]
 wasm-bindgen = { path = "../..", version = "0.2.76" }
+num-bigint = { version = "0.4.1", optional = true }
+num-traits = { version = "0.2.14", optional = true }
```

Comment:
Sorry but personally I do not want to add new public dependencies to these crates. There is no story for landing breaking changes to js-sys at this time and having new public dependencies increases the risk that a breaking change may be needed (in case these deps have a major-version-bump)

I realize these are optional but it's still part of the public API and something that I'm not willing to commit to.